src/modules/scores.js

```diff
@@ -1,7 +1,30 @@
-import Score from './score.js';
-
 export default class Scores {
-  static scores = [new Score('Aaron', 2000), new Score('Juan', 1850), new Score('Juda', 2175)];
+  static scoreSource = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/2XZBFeB4G4BRrQVtaC1y/scores';
+
+  static scores = [];
+
+  static getScores = () => this.scores;
+
+  static getAPI = async () => {
+    const response = await fetch(this.scoreSource);
+    const values = await response.json();
+    return values.result;
+  }
+
+  static postScore = async (userName, score) => {
+    const response = await fetch(this.scoreSource, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json; charset=UTF-8',
+      },
+      body: JSON.stringify({
+        user: `${userName}`,
+        score: `${score}`,
+      }),
+    });
 
-  static getScores = () => Scores.scores;
+    const data = await response.json();
+    console.log(data);
```

Comment:
- [ ] Please, kindly delete this debugging `console.log` statement from the repo, according to the Javascript required best practices.

Reply:
@Meltrust 

Oops. Got it, thanks